polling_stations/apps/data_finder/directions_clients.py

```diff
@@ -0,0 +1,129 @@
+import abc
+import json
+import requests
+import urllib
+from collections import namedtuple
+from django.conf import settings
+from django.utils.translation import ugettext as _
+
+
+Directions = namedtuple('Directions', [
+    'walk_time', 'walk_dist', 'route', 'precision', 'source'])
```

Comment:
Future proofing question: should we add the transport method here, and rename `walk_[time|distance]` to `[time|distance]`? If we want to add driving times in the future (for example, when the `walk_time` is greater than ~15 minutes) then this namedtuple isn't useful.